src/resources/org/apache/cassandra/cql3/reserved_keywords.txt

```diff
@@ -54,4 +54,7 @@ USE
 USING
 VIEW
 WHERE
-WITH
\ No newline at end of file
+WITH
+CONSTRAINT
+LENGTH
+CHECK
```

Comment:
Can they be unreserved? Since the constraints are defined starting with "CONSTRAINT", I think you can leave the other 2 out as normal keywords.

Reply:
Internal testing complaints if they are not reserved, as they appear set as Keywords on the lexer